nixos/modules/misc/ids.nix

```diff
@@ -301,6 +301,7 @@
       pykms = 282;
       kodi = 283;
       restya-board = 284;
+      tt_rss = 285;
```

Comment:
The application directory is `chowned` on startup. Therefor it is not necessary to allocate a static uid/gid.

Reply:
But tt_rss must be given some uid/gid somewhere anyway, right? If not here, then the user must provide it. At least that's how it works with immutable users and I'd prefer supporting immutable users seamlessly.

Reply:
No, the user must not provide it. It will be allocated on the first use.
Static uids/gids are a limited resource. Therefore we only allocate them, if necessary.

Reply:
Oh, will it be allocated even with immutable users?

Reply:
Do you have any example user ID's or services in your mind where no static uids/gids were used but automatically generated?

Reply:
If it is an issue that each webapp defines and uses its own uid, we could add a generic `phpfpm` uid which can be used by all webapps using php-fpm. Similarly, as `uwsgi` uid. But of course it's best if each webapp is "isolated" with its own uid.